misc/vim/syntax/dbscheme.vim

```diff
@@ -0,0 +1,11 @@
+syn keyword dbschemeKeyword case of
+syn keyword dbschemeType int ref string varchar 
+
+syn match dbschemeCustomType "\v[@][a-zA-Z_0-9]+" 
+
+syn region dbschemeComment start='/\*' end='\*/'
```

Comment:
Can you also add support for comments starting with `//`?

Reply:
Done!